xds/src/main/java/io/grpc/xds/XdsNameResolver.java

```diff
@@ -217,6 +221,9 @@ public void start(Listener2 listener) {
     ldsResourceName = XdsClient.canonifyResourceName(ldsResourceName);
     callCounterProvider = SharedCallCounterMap.getInstance();
     resolveState = new ResolveState(ldsResourceName);
+    ServerInfo serverInfo = xdsClient.getServerInfo(ldsResourceName);
```

Comment:
This is not compatible with fallback. With fallback, each authority will have multiple server infos. We have to use the Args passed to the resource parser.